README.md

```diff
@@ -95,6 +95,14 @@ You will need to set up headers with both:
 
 _If you're new to GraphQL, you can checkout [Artsy's GraphQL Workshop](https://github.com/artsy/graphql-workshop)._
 
+### Introspection Setup
```

Comment:
What do you think about mentioning that schema documentation from staging or production endpoints is only available for internal use.

In development (where `INTROSPECT_TOKEN` is not usually set), you don't need to send this header to fetch schema docs.